### DISCUSSION TITLE
Utility and Output Voltage

### DISCUSSION THREAD

**Discussion by froussy** [jgyates/genmon#1176] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).
- [X] I have included the generator make and model in my submission.

### Version

1.19.04 / Evo 2 controller

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have submitted logs

### What is/are your question(s)?

Good morning,

First of all, want to say that it's a good and and work well :)

I am wondering one thing..   In the monitor, there is "Utility Voltage", and "Output Voltage".

I know one is the voltage from the generator, and the other from the power utility company.

Actually, my Voltage In is 208v and the generator output 220v, which is fine.

In the config, there is a nominallinevolts setting to adapt the graph for the right voltage.. but changing it to 208v, change both graph.. 

Is there another variable to adjust both differently?

Thanks

**Comment by jgyates** [comment]
Hi @froussy

No, current the graph and gauge adjustment for the nominal line voltage setting applies to both generator output and utility voltage.

The only thing that this will effect is the center values of the gauges. You could put the nominal value to 214.

**Comment by froussy** [comment]
Hi jgyates

 I just tried with the 214 value. that will do the job

thanks!! :)
